keps/prod-readiness/sig-scheduling/3521.yaml

```diff
@@ -0,0 +1,3 @@
+kep-number: 3521
+alpha:
+  approver: "@wojtek-t"
```

Comment:
@johnbelamaric can you take this one?

Reply:
@wojtek-t will you be able to take this?

Reply:
Yes - will get to it later this week.